cli/Sources/ProjectDescription/CacheOptions.swift

```diff
@@ -1,13 +1,28 @@
 extension Tuist {
     /// Options for caching.
     public struct CacheOptions: Codable, Equatable, Sendable {
+        /// When true, it preserves the sources of all the targets in the graph. Note that when this option is set to true, the
+        /// graph is not focused.
         public var keepSourceTargets: Bool
 
+        /// You can use this option to limit the network concurrency at the application level. By default is is set to 15, and you
+        /// can use `nil` to drop the limit.
+        public var concurrencyLimit: Int?
```

Comment:
I'm not sure if this is something we want to expose users to. I wonder if it's better to make the `concurrencyLimit` configurable just via env variables and once we validate we don't need to set a concurrency limit, we'd drop the env variable again.

Fwiw, I don't necessarily mind this variable _right now_, but I think we should aim to _eventually_ remove it – whereas as-is, teams would probably expect us to keep this variable forever, which I'm not sure we want to commit to.